src/tls.c

```diff
@@ -2948,6 +3026,24 @@ static word16 TLSX_EllipticCurve_WriteR(EllipticCurve* curve, byte* output)
     return OPAQUE16_LEN + offset;
 }
 
+#endif
+
+static word16 TLSX_PointFormat_WriteR(PointFormat* point, byte* output);
+static word16 TLSX_PointFormat_WriteR(PointFormat* point, byte* output)
```

Comment:
Will you refactor the `TLSX_PointFormat_WriteR` and  `TLSX_EllipticCurve_WriteR` functions to not use recursive calls (use a while loop instead). Also you can remove the duplicate static function declaration for both; those are not required.